props/saber_fett263_buttons.h

```diff
@@ -45,14 +78,51 @@ SaberFett263Buttons() : PropBase() {}
   bool swinging_ = false;
   void Loop() override {
     PropBase::Loop();
+    // Swing On gesture control this portion allows fine tuning of speed needed to ignite
+    if(!SaberBase::IsOn() ) {
+      if (millis() - saber_off_time_ < MOTION_TIMEOUT) {
+        SaberBase::RequestMotion();
+      // Edit '250' value in line below to change swing on sensitivity, 250 ~ 400 work best in testing
+        if (!swinging_ && fusor.swing_speed() > 250) {
+          swinging_ = true;
+          Event(BUTTON_NONE, EVENT_SWING);
+        }
+        if (swinging_ && fusor.swing_speed() < 100) {
+          swinging_ = false;
+        }
+      }
+    }
+	  
     if (!swinging_ && fusor.swing_speed() > 250) {
       swinging_ = true;
       Event(BUTTON_NONE, EVENT_SWING);
     }
+	  
+    if (auto_lockup_on_) {
+      if (!swinging_ && fusor.swing_speed() > 120 && fusor.swing_speed() < 250 && millis() - clash_impact_millis_ > FETT263_LOCKUP_DELAY) {
+        if (SaberBase::Lockup()) {
+          SaberBase::DoEndLockup();
+          SaberBase::SetLockup(SaberBase::LOCKUP_NONE);
+          auto_lockup_on_ = false;        
+        } 
+      }
+    }
+	  
     if (swinging_ && fusor.swing_speed() < 100) {
+      last_swing_ = millis();
       swinging_ = false;
+    }   
+    
+    if (auto_melt_on_) {
+      if (!swinging_ && fusor.swing_speed() > 60 && fusor.swing_speed() < 120 && millis() - clash_impact_millis_ > FETT263_LOCKUP_DELAY) {  
```

Comment:
this line is too long, I suggest breaking it up like:

 if (!swinging_ &&
     fusor.swing_speed() > 60 &&
     fusor.swing_speed < 120 &&
     millis() - clash_imoact_millis_ > FETT263_LOCKUP_DELAY) {
      ....
 }

also, if (A) {
      if (B) {

can be written as:

   if (A && B)

so, it should probably be:

   if (auto_melt_on_ &&
        !swinging_ &&
        ....